cmd/installer/cli/install.go

```diff
@@ -350,25 +350,16 @@ func addInstallAdminConsoleFlags(cmd *cobra.Command, flags *InstallCmdFlags) err
 
 func addManagementConsoleFlags(cmd *cobra.Command, flags *InstallCmdFlags) error {
 	cmd.Flags().IntVar(&flags.managerPort, "manager-port", ecv1beta1.DefaultManagerPort, "Port on which the Manager will be served")
-	cmd.Flags().StringVar(&flags.tlsCertFile, "tls-cert", "", "Path to the TLS certificate file")
-	cmd.Flags().StringVar(&flags.tlsKeyFile, "tls-key", "", "Path to the TLS key file")
+	cmd.Flags().StringVar(&flags.tlsCertFile, "tls-cert", "", "Path to the TLS certificate file for the Admin Console")
+	cmd.Flags().StringVar(&flags.tlsKeyFile, "tls-key", "", "Path to the TLS key file for the Admin Console")
```

Comment:
is `Admin Console` the right term for both types of installs?

Reply:
@ajp-io any suggestions to what the copy here should be?

Reply:
we've used the manage wording before, though i don't love that word either. might need to talk to kaylee about how should refer to this thing.

but we could just have different wording here for v3 vs. v2 and say admin console for v2 and installer for v3. i don't think "for the manager" will make much sense to people.

Reply:
we use Manager everywhere else. should we change it elsewhere?

Reply:
on line 352

```
	cmd.Flags().IntVar(&flags.managerPort, "manager-port", ecv1beta1.DefaultManagerPort, "Port on which the Manager will be served")
```